### PR TITLE
Add Open at Login

### PR DESCRIPTION
**Pull request by leeknowlton**
- Add "Open at Login" feature
- Modify settings menu to allow for open at login toggle

Response to #145

### CODE REVIEW

File: src/renderer/components/settings.tsx [hunk 1]
```diff
@@ -118,12 +124,63 @@ export default function SettingsMenu({
...
+	useEffect(() => {
+		const fetchOpenAtLogin = async () => {
+			const isOpen = await settings.getOpenAtLogin();
+			setOpenAtLogin(isOpen);
+		};
+		fetchOpenAtLogin();
+	}, []);
+
+	useEffect(() => {
+		openAtLogin
```
Comment (swyxio) [inline src/renderer/components/settings.tsx:136]: can you add this in the dependency array please?

**Review by swyxio — CHANGES_REQUESTED** [review]
wow awesome thanks for getting this done! just 1 tweak so far (reviewing on my phone haha)

File: src/renderer/components/settings.tsx (hunk 1 above)
Comment (leeknowlton) [reply src/renderer/components/settings.tsx:136]: Added 👍

**Comment by leeknowlton** [comment]
This is my first public repo PR (at least first in a long time!). Not sure if notifications appear for new commits, but made the changes you mentioned.

**Comment by swyxio** [comment]
yes appreciated! i just needed some time since i was traveling this week

**Comment by swyxio** [comment]
tweaked wording, added to readme, added a lil image for reference

image

**Pull request merged**